electron-builder-config.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs');
+const { execFileSync } = require('child_process');
+const pkg = require('./package');
+
+const OS = process.platform;
+const ARCH = process.arch;
+const EXT = OS === 'win32' ? 'exe' : 'dmg';
```

Comment:
Would this mean that the appImage on Linux (since there's a configuration defined below) also have the `.dmg` extension?

Reply:
Thanks for pointing this out! I opted to remove the linux section of the config as the rest of our build process (specifically the invest side) doesn't support linux right now.

Fun fact, one thing we lost when moving this config out of `package.json` were some built-in template variables for these strings. So now we have to make them ourselves, but no big deal.

Reply:
Oh that's interesting about the template variables being available in `package.json` but not here.  The approach you have here makes sense to me.